vs-code-home-page-samples/data-mapper/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "praneesha"
```

Comment:
Lets change org to `integration_samples`

Reply:
Fixed.